src/config/nunjucks/context/context.test.js

```diff
@@ -3,11 +3,11 @@ import { beforeAll, beforeEach, describe, expect, test, vi } from 'vitest'
 const mockReadFileSync = vi.fn()
 const mockLoggerError = vi.fn()
 
-vi.mock('node:fs', async () => ({
+vi.mock(import('node:fs'), async () => ({
```

Comment:
Separate to this PR I suppose, but we have a lot of mocks of code we don't own. Might be nice to get some ports & adapters in at some point.